PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -429,6 +429,10 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64 };
 	// Users should call it with pluginRootPath be NULL to get the required number of TCHAR (not including the terminating nul character),
 	// allocate pluginRootPath buffer with the return value + 1, then call it again to get the path.
 
+	#define NPPM_UPDATECLICKABLELINKS (NPPMSG + 98)

```

Comment:
Please define rather `NPPM_INTERNAL_UPDATECLICKABLELINKS` in `\PowerEditor\src\resource.h`.
NPPM_MESSAGES in `PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h` are the public messages (API) for plugins use.